### ISSUE TITLE
Crash when fetching video information

### ISSUE THREAD

**Issue by shadone** [SvenTiigi/YouTubePlayerKit#79]
### What happened?

The crash occurs on [YouTubePlayerWebView+Evaluate.swift:147](https://github.com/SvenTiigi/YouTubePlayerKit/blob/94b5739d32f0108f1820091e6955d544a0363fad/Sources/WebView/YouTubePlayerWebView%2BEvaluate.swift#L147) on line `.store(in: &self.cancellables)`

When it crashes, the execution happens in background thread, I believe the issue is concurrent access to shared variable `cancellables`.

I am not sure that's the best solution, but adding `@MainActor` attribute to `getInformation()` solves the problem for me 

```swift
    #if compiler(>=5.5) && canImport(_Concurrency)
    /// Retrieve the YouTubePlayer Information
    @MainActor // <<<< this solves the problem for me.
    func getInformation() async throws -> Information {
        try await withCheckedThrowingContinuation { continuation in
            self.getInformation(completion: continuation.resume)
        }
    }
    #endif
```

### What are the steps to reproduce?

Here is a piece of code showing how I use YouTubePLayerView

```swift
struct VideoView: View {
    let player: YouTubePlayer

    @State private var title: String?
    @State private var duration: Double?

    init(youtubeUrl: URL) {
        var config = YouTubePlayer.Configuration()
        config.fullscreenMode = .system
        config.useModestBranding = true
        config.playInline = false

        player = YouTubePlayer(source: .url(youtubeUrl.absoluteString), configuration: config)
    }

    var body: some View {
        VStack {
            YouTubePlayerView(player)
                .frame(height: 192)
            if let title {
                Text(title)
                    .multilineTextAlignment(.leading)
                    .padding(EdgeInsets(top: 32, leading: 16, bottom: 0, trailing: 16))
                    .frame(maxWidth: .infinity, alignment: .leading)
            }
            Button("\(duration ?? 0)", systemImage: "play.fill") { }
                .buttonStyle(.play)
                .padding(EdgeInsets(top: 0, leading: 16, bottom: 16, trailing: 16))
                .frame(maxWidth: .infinity, alignment: .leading)
        }
        .task { @MainActor in
            async let info = player.getInformation()
            async let metadata = player.getPlaybackMetadata()
            title = try? await metadata.title
            duration = try? await info.duration
        }
    }
```

### What is the expected behavior?

Not crash would be great :)

**Comment by SvenTiigi** [comment]
Hi @shadone,

Thanks for your detailed bug report 👍

A [fix](https://github.com/SvenTiigi/YouTubePlayerKit/commit/928fbadbd8a6f4a50eb4be85251e5b175a23dfac) is now available on the [develop branch](https://github.com/SvenTiigi/YouTubePlayerKit/tree/develop).
Please let me know if this resolves your issue.

**Comment by shadone** [comment]
I believe that solves it! I cannot reproduce the crash any more. Thanks for such a quick reply!

**Comment by SvenTiigi** [comment]
The fix is now available with the latest release [1.6.0](https://github.com/SvenTiigi/YouTubePlayerKit/releases/tag/1.6.0)
